input/input.md

```diff
@@ -196,17 +196,27 @@ An example of the `element_testing/inputFiles/mesh.constraints` is shown below.
 In this case, there are 4 general constraints set on nodes and 4 frictional constraints. Multiple constraints can be set on nodes, for instance: `Node 0` has constraints in both the `0` direction and the `1` direction. 
 
 
-## Soil Particles (Points)
+## Soil Particles (Points) File
 
-Then, we need points as after all the method is called material point method for a reason. The initial location of the points should not coincide with the nodes of the mesh, neither should it be on the boundary of the elements.
+The the geometry (location) of points to be used in the model must be set in a file pointed to by the `inputSoilParticleFileName` header of the `input.dat` file. 
```

Comment:
Repeating `the`, and `The location of material points are specified in the 'inputSoilParticleFileName'`